code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer //designed to be a very forgiving and standard abnormality.
+	name = "forsaken murderer"
+	desc = "A unhealthy looking human bound in a full body straightjacket. His neck is broken and in the middle of his forehead is a old wound that refuses to heal."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "forsakenmurdererinert"
+	icon_living = "forsakenmurdererinert"
+	icon_dead = "forsakenmurdererdead"
+	del_on_death = FALSE
+	maxHealth = 540 //originally was 270
+	health = 540
+	rapid_melee = 1 //attack speed modifier. 2 is twice the normal.
+	move_to_delay = 5 //speed? lower is faster.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 4 //the lowest damage in regular attacks. Normal murderer is 2~4 so we double it.
+	melee_damage_upper = 8
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE //is the second half of melee damage type. This the armor type checked when attacking someone.
+	stat_attack = CONSCIOUS //originally was HARD_CRIT. This setting makes forsaken murderer attack you until you go into crit.
+	attack_sound = 'sound/effects/hit_kick.ogg' //used chrome to listen to these
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	faction = list("hostile") //hostile faction makes them friendly with most abnormalities. A choice i dont entirely like.
+	can_breach = TRUE
+	threat_level = TETH_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(60, 60, 50, 50, 50),
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 30, 30, 30),
+						ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 40, 40, 40),
+						ABNORMALITY_WORK_REPRESSION = list(30, 20, 0, -80, -80),
+						)
+	work_damage_amount = 6
+	work_damage_type = RED_DAMAGE
+	deathmessage = "falls over."
+	environment_smash = 1
+	speak_chance = 2
+	emote_see = list("shakes while mumbling...")
+	emote_hear = list("screams!")
+	max_boxes = 14
+	//success_boxes = 7 //said to be a undefined var
+	zone_selected = BODY_ZONE_CHEST //alternate is BODY_ZONE_HEAD but this made damage go past the armor
+	speed = -1 //actual speed?
+
+	ego_list = list(
+		/datum/ego_datum/weapon/regret,
+		/datum/ego_datum/armor/regret
+		)
+
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer/failure_effect(mob/living/carbon/human/user, work_type, pe) //when work type is bad the qliphoth counter lowers with no chance.
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer/death(gibbed) //So people see him fall. Cant seem to get the death animation to play.
```

Comment:
It(the death animation) doesn't work because of the update_icon_state override you've done below.

Reply:
Dont i need that update_icon_state() for the beaching sprite to change?

Reply:
You don't *really* need to use update_icon at all, given you simply change the icon_state.